src/Faker/Provider/DateTime.php

```diff
@@ -33,34 +34,42 @@ public static function unixTime($max = 'now')
     }
 
     /**
-     * Get a datetime object for a date between January 1, 1970 and now
+     * Get a datetime object for a date between January 1, 1970 and now.
      *
      * @param \DateTime|int|string $max maximum timestamp used as random end limit, default to "now"
+     *
      * @example DateTime('2005-08-16 20:39:21')
+     *
      * @return \DateTime
      */
     public static function dateTime($max = 'now')
     {
-        return new \DateTime('@' . static::unixTime($max));
+        return new \DateTime('@'.static::unixTime($max));
     }
 
     /**
-     * Get a datetime object for a date between January 1, 001 and now
+     * Get a datetime object for a date between January 1, 001 and now.
      *
      * @param \DateTime|int|string $max maximum timestamp used as random end limit, default to "now"
+     *
      * @example DateTime('1265-03-22 21:15:52')
+     *
      * @return \DateTime
      */
     public static function dateTimeAD($max = 'now')
     {
-        return new \DateTime('@' . mt_rand(-62135597361, static::getMaxTimestamp($max)));
```

Comment:
This method already works and is much faster than your `dateTimeBetween()` implementation. So your change just makes it slower. Please revert.